core/templates/pages/exploration-editor-page/exploration-editor-page.component.html

```diff
@@ -14,7 +14,7 @@ <h3 class="e2e-test-joyride-title">Preview</h3>
     <div class="navbar-icons">
       <li [ngClass]="{'navbar-tab-active': getActiveTabName() === 'main', 'uib-dropdown': countWarnings()}" (click)="selectMainTab()"
           class="nav-item icon nav-list-item position-relative">
-        <a href="#" class="nav-link navbar-tab e2e-test-main-tab" [ngbTooltip]="'Editor'" placement="{{countWarnings() ? 'left' : 'bottom'}}">
+        <a href="#" class="nav-link navbar-tab e2e-test-main-tab" [ngbTooltip]="'Editor'" placement="{{countWarnings() ? 'left' : 'bottom'}}" aria-label="Editor" >
```

Comment:
@hemantmm Why do you have all these additions? Could you please demonstrate what Chromevox says before and after the change?

Reply:
@seanlip While I was solving another issue by mistake the exploration file got committed in this issue, so I deleted the file.